experimenter/experimenter/experiments/api/v5/serializers.py

```diff
@@ -897,6 +898,9 @@ class NimbusExperimentSerializer(
         allow_null=True,
         required=False,
     )
+    qa_comment = serializers.CharField(
+        min_length=0, max_length=1024, required=False, allow_blank=True
```

Comment:
Might wanna bump this up to like 4096 just to be safe, sometimes people can put a lot of info in there.